manufacturing-ai-suite/industrial-edge-insights-multimodal/docs/user-guide/how-to-guides/how-to-update-config.md

```diff
@@ -1,8 +1,8 @@
 # Update config in Time Series Analytics Microservice
 
-The Time Series Analytics Microservice provides an interactive Swagger UI at `http://<host_ip>:5000/docs`.
+The Time Series Analytics Microservice provides an interactive Swagger UI at `https://<host_ip>:5000/docs`.
 
-> **Note:** Use the link `http://<host_ip>:30001/ts-api/docs` to access the
+> **Note:** Use the link `https://<host_ip>:30001/ts-api/docs` to access the
 > Swagger UI if doing a Helm-based deployment on a Kubernetes cluster.
```

Comment:
We can remove this note as we aren't providing the helm charts now, we can bring it back once we provide the helm charts